src/app-layout/drawer/interfaces.ts

```diff
@@ -45,8 +45,6 @@ export interface ResizableDrawerProps extends DesktopDrawerProps {
 }
 
 export interface DrawerTriggersBarProps {
-  contentClassName: string;
-  toggleClassName: string;
```

Comment:
These never change, so not much of a point to make them a props. Fewer props make it easier to understand components